libcob/intrinsic.c

```diff
@@ -1246,7 +1246,7 @@ cob_intr_date_of_integer (cob_field *srcdays)
 	int		leapyear = 365;
 	cob_field_attr	attr;
 	cob_field	field;
-	char		buff[16];
+	char		buff[25];
```

Comment:
Even if the size of buff is 24, the warning messages will be solved.
Set the size of buff to 24.